docs/topics/tech-tags.adoc

```diff
@@ -2,7 +2,7 @@
 //
 // * docs/cli-guide/master.adoc
 
-[id='tech-tags_{context}']
+[id="tech-tags_{context}"]
 = Supported Technology Tags
```

Comment:
Use sentence case. "Technology Tags" can be used generically. It looks odd to have it capitalized, so I would lower case it except when referring to the UI.